requirements/main.in

```diff
@@ -39,7 +39,7 @@ paginate_sqlalchemy
 passlib>=1.6.4
 pip-api
 premailer
-psycopg2
+psycopg
```

Comment:
```suggestion
psycopg[c]
```

Per https://www.psycopg.org/psycopg3/docs/basic/install.html#local-installation

Unless we want to forego compilation, and use `psycopg[binary]` for pre-compiled. Otherwise we get a [Pure Python installation](https://www.psycopg.org/psycopg3/docs/basic/install.html#pure-python-installation)

Reply:
Given "This is the preferred way to install Psycopg for a production site.", seems to be the right choice.